backend.native/tests/interop/objc/smoke.m

```diff
@@ -76,3 +76,9 @@ -(void)dealloc {
   unexpectedDeallocation = YES;
 }
 @end;
+
+@implementation BlockCaller
++(void)callBlock:(void (^)(void))block {
+    block();
```

Comment:
What we gonna do for `staticCFunction`?

Reply:
Could you clarify? Both blocks and C callbacks are allowed to throw Kotlin exceptions.

Reply:
I mean, maybe test both?